### ISSUE TITLE
3rd party finance liability transfer needs record in FINCO2FI_SO added

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2134]
_Imported from trac ticket 2134.
Created by **sal**
Opened in trac: Feb 11, 2013, 9:47:43 AM
Last modified in trac: Feb 20, 2013, 10:57:41 AM_
***
_by sal:_
This is needed for FCRIN screen.

**Comment by kentbower** [comment]
_**kbower** trac comment on Feb 11, 2013, 11:59:43 AM:_

Wayne,
    When a liability transfer to a third party financing company occurs for an order, there are 2 dollar balancing transfers AR_TRN records that are written.  Apparently there is data that must be inserted into FINCO2FI_SO, correct?

1. Exactly what needs to be inserted into this table?
2. Are there any other events that Loft must also perform?

Thanks very much for your expertise!

Kent

**Comment by kentbower** [comment]
_**kbower** trac comment on Feb 11, 2013, 12:03:27 PM:_

```
  CREATE TABLE "AR"."FINCO2FI_SO"
   (    "CO_CD" VARCHAR2(3) NOT NULL ENABLE,
        "FINCO_CUST_CD" VARCHAR2(10) NOT NULL ENABLE,
        "IVC_CD" VARCHAR2(14) NOT NULL ENABLE,
        "ACCT_NUM" VARCHAR2(20),
        "CUST_CD" VARCHAR2(10) NOT NULL ENABLE,
        "FUND_DT" DATE NOT NULL ENABLE,
         CONSTRAINT "FINCO2FI_SO_PK" PRIMARY KEY ("CO_CD", "FINCO_CUST_CD", "IVC_CD")
  USING INDEX PCTFREE 10 INITRANS 30 MAXTRANS 255  NOLOGGING
  STORAGE(INITIAL 40960 NEXT 1048576 MINEXTENTS 1 MAXEXTENTS 2147483645
  PCTINCREASE 0 FREELISTS 22 FREELIST GROUPS 1 BUFFER_POOL DEFAULT)
  TABLESPACE "INDEXES"  ENABLE,
         CONSTRAINT "FINFINSO_CUST_FKD" FOREIGN KEY ("CUST_CD")
          REFERENCES "AR"."CUST" ("CUST_CD") DISABLE,
         CONSTRAINT "FINFINSO_CUST_FKD2" FOREIGN KEY ("FINCO_CUST_CD")
          REFERENCES "AR"."CUST" ("CUST_CD") DISABLE
   ) PCTFREE 10 PCTUSED 40 INITRANS 30 MAXTRANS 255 NOCOMPRESS LOGGING
  STORAGE(INITIAL 16384 NEXT 1048576 MINEXTENTS 1 MAXEXTENTS 2147483645
  PCTINCREASE 0 FREELISTS 22 FREELIST GROUPS 1 BUFFER_POOL DEFAULT)
  TABLESPACE "TABLES"

```

**Comment by kentbower** [comment]
_**kbower** trac comment on Feb 12, 2013, 12:36:25 PM:_

___
Subject:         RE: #2134: Liability transfer
Date:         Tue, 12 Feb 2013 17:04:46 +0000
From:         Wayne Liburd <wliburd@kanesfurniture.com>
To:         Kent Bower <kb@retailarchitects.com>



Kent,

Good morning, yes that is correct. The company code needs to be inserted CO_CD, the finance company code into FINCO_CUST_CD, the sales order number needs to inserted into IVC_CD, the account number needs to be inserted into ACCT_NUM, the customer code into CUST_CD and the written date into FUND_DT.

I will give that some more thought and get back to you.

Thanks,
 Wayne Liburd

**Comment by kentbower** [comment]
_**rneville** trac comment on Feb 13, 2013, 12:48:08 PM:_

(Changeset [[3686]](../commit/71ebc2b263d336f5495f50eae50c96683969b3fa))
* fixes #2134 (3rd party finance liability transfer needs record in FINCO2FI_SO added)

**Comment by kentbower** [comment]
_**rneville** trac status change on Feb 13, 2013, 12:48:08 PM:_ **closed**

**Comment by kentbower** [comment]
_**rneville** trac comment on Feb 13, 2013, 1:01:36 PM:_

(Changeset [[3687]](../commit/0bab192782c58eed4536f9e287d6ff42205d7d7b))
* quick fix #2134

**Comment by kentbower** [comment]
_**rneville** trac comment on Feb 20, 2013, 10:57:41 AM:_

(Changeset [[3722]](../commit/3787566b65a86bc74913c33661d575ef86e2b49f))
* references #2134 - additional column and logic needed
